docs/pages/database-access/reference/cli.mdx

```diff
@@ -105,6 +105,7 @@ $ teleport db configure create \
 | `--token` | Invitation token to register with the Auth Service. Default: none. |
 | `--rds-discovery` | List of AWS regions the agent will discover for RDS/Aurora instances. |
 | `--redshift-discovery` | List of AWS regions the agent will discover for Redshift instances. |
+| `--elasticache-discovery` | List of AWS regions the agent will discover for ElastiCache Redis clusters. |
```

Comment:
```suggestion
| `--elasticache-discovery` | List of AWS regions in which the agent will discover ElastiCache Redis clusters. |
```

Reply:
I've updated the original usage in the binary and updated the reference here.